src/callbacks/ContentEditableChange.js

```diff
@@ -0,0 +1,8 @@
+const isContentEditableChildList = (mutation) =>
+  mutation.type === 'childList' &&
+  mutation.target.hasAttribute('contentEditable');
+
+export const handleContentEditableChange = (action) => (mutations) =>
+  mutations
+    .filter(isContentEditableChildList)
+    .forEach((mutation) => action(mutation.target));
```

Comment:
I love that you split these callbacks out and added separate tests for them.  It provides a great improvement to clarity.

Reply:
Thanks! Was a breakthrough moment during the refactor! 😄